src/pages/instances/InstanceSearchFilter.tsx

```diff
@@ -56,24 +49,71 @@ const InstanceSearchFilter: FC<Props> = ({ instances, setFilters }) => {
       }),
     },
   ];
-
+  const setSearchParams = useSearchParams()[1];
   const onSearchDataChange = (searchData: SearchAndFilterChip[]) => {
-    setFilters({
-      queries: searchData
-        .filter((chip) => chip.quoteValue)
-        .map((chip) => chip.value.toLowerCase()),
-      statuses: enrichStatuses(
-        searchData
-          .filter((chip) => chip.lead === "status")
-          .map((chip) => chip.value as LxdInstanceStatus),
-      ),
-      types: searchData
-        .filter((chip) => chip.lead === "type")
-        .map((chip) => (chip.value === "VM" ? "virtual-machine" : "container")),
-      profileQueries: searchData
-        .filter((chip) => chip.lead === "profile")
-        .map((chip) => chip.value),
+    const filteredQueries = searchData
+      .filter((chip) => chip.quoteValue === true || chip.lead === "query")
+      .map((chip) => chip.value.toLowerCase());
+
+    const filteredStatuses = searchData
+      .filter((chip) => chip.lead === "status")
+      .map((chip) => chip.value as LxdInstanceStatus);
+    const filteredTypes = searchData
+      .filter((chip) => chip.lead === "type")
+      .map((chip) => (chip.value === "VM" ? "virtual-machine" : "container"));
```

Comment:
Rewriting it here has a wired side effect: When selecting VM type and reloading the page, we get containers instead of VMs. I think we can fix it by:
1. keep it here as `.map((chip) => chip.value)`.
2. move the rewrite to InstanceList: `types: searchParams.getAll("type").map((value) => value === "VM" ? "virtual-machine" : "container")`